css/styles.css

```diff
@@ -497,10 +497,6 @@ section#developers .section-header {
 
```

Comment:
And to answer your question, yes I think so, we should copy as much as possible but add openhab specific things on top. A styles2.css loaded after style.css could be a place for everything orange and openhab'y.